src/mito2/src/region/opener.rs

```diff
@@ -54,6 +54,7 @@ pub(crate) struct RegionOpener {
     scheduler: SchedulerRef,
     options: HashMap<String, String>,
     cache_manager: Option<CacheManagerRef>,
+    skip_replay_wal: bool,
```

Comment:
skip_wal_replay? `replay` can be a noun.

Reply:
Good point